docs/validators/staking-v4.md

```diff
@@ -0,0 +1,297 @@
+be 1216 such that:
+
+![](/validators/stakingV4/soft-auction2.png)
+
+```
++--------+------------------+----------------+--------------+-------------------+-----------------------------+------------------+---------------------------+-----------------------------+
+| Owner  | Num staked nodes | TopUp per node | Total top up | Num auction nodes | Num qualified auction nodes | Num active nodes | Qualified top up per node | Selected auction list nodes |
++--------+------------------+----------------+--------------+-------------------+-----------------------------+------------------+---------------------------+-----------------------------+
+| owner1 | 3                | 1222           | 3666         | 1                 | 1                           | 2                | 1222                      | pubKey1                     |
+| owner2 | 3                | 851            | 2555         | 2                 | 1                           | 1                | 1277                      | pubKey2                     |
+| owner3 | 2                | 1222           | 2444         | 1                 | 1                           | 1                | 1222                      | pubKey4                     |
+| owner4 | 4                | 666            | 2666         | 3                 | 1                           | 1                | 1333                      | pubKey5                     |
++--------+------------------+----------------+--------------+-------------------+-----------------------------+------------------+---------------------------+-----------------------------+
+```
+
+- `owner1` has 1 auction node: `pubKey1` with qualified topUp per node = 1222 > 1216
+- `owner2` has 2 auction nodes: `pubKey2`, `pubKey3` with a topUp per node = 851. By leaving one of his nodes in
+  auction(`pubKey3`), while only selecting one of them(`pubKey2`), his topUp per node would be rebalanced to
+  1277 (2555/2) > 1216
+- `owner3` has 1 auction node: `pubKey4` with qualified topUp per node = 1222 > 1216
+- `owner4` has 3 auction nodes: `pubKey5`, `pubKey6`, `pubKey7` with a topUp per node = 666. By leaving two of his
+  nodes in auction(`pubKey6`, `pubKey7`), while only selecting one of them(`pubKey5`), his topUp per node would be
+  rebalanced to 1333 (2666/2) > 1216
+
+If we had increased the threshold above by one more step from `1216` to `1226`, we would only have two qualified
+nodes: `pubKey2` and `pubKey5`, which is not enough to fill all slots.
+
+:::note
+
+If an owner has multiple nodes in auction, but only a part of them are selected to be distributed in waiting list,
+selection will be done by sorting the bls keys.
+
+:::
+
+:::note
+
+Minimum required topUp per node, as well as real time auction list will be available in explorer at all times so that
+owners can decide what's the best strategy to maximize the number of auction nodes that will be selected.
```

Comment:
```suggestion
The minimum required topUp per node, as well as real time auction list, will be public in explorer at all times so that owners can decide the best strategy to maximize the number of auction nodes selected.
```